plugins/slicer/MONAILabelReviewer/MONAILabelReviewerLib/ImageDataController.py

```
@@ -237,6 +237,15 @@ def reuqestSegmentation(self, image_id: str, tag: str) -> requests.models.Respon
     def getDicomDownloadUri(self, image_id: str) -> str:
         return self.monaiServerREST.getDicomDownloadUri(image_id)
 
+    def requestImage(self, image_id: str):
+        if img_blob is not None:
```

Ruff (0.16.0)

[error] Undefined name `img_blob`

(F821)